Robust.Server/Robust.Server.csproj

```
@@ -13,12 +13,24 @@
     <PackageReference Include="JetBrains.Annotations" Version="2021.3.0" PrivateAssets="All" />
     <PackageReference Include="SpaceWizards.HttpListener" Version="0.1.0" />
     <!--  -->
-    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.7" />
     <PackageReference Include="prometheus-net" Version="4.1.1" />
     <PackageReference Include="Serilog.Sinks.Loki" Version="4.0.0-beta3" />
     <PackageReference Include="Microsoft.Extensions.Primitives" Version="6.0.0" />
     <PackageReference Include="prometheus-net.DotNetRuntime" Version="4.2.2" />
   </ItemGroup>
+  <Choose>
+    <When Condition="'$(UseSystemSqlite)' == 'True'">
+      <ItemGroup>
+        <PackageReference Include="Microsoft.Data.Sqlite.Core" Version="6.0.8" />
+        <PackageReference Include="SQLitePCLRaw.provider.sqlite3" Version="2.1.1" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.0.7" />
+      </ItemGroup>
+    </Otherwise>
+  </Choose>
```

Simpler way to do it.

```suggestion
    <PackageReference Include="Microsoft.Data.Sqlite.Core" Version="6.0.9" />
    <PackageReference Include="SQLitePCLRaw.provider.sqlite3" Version="2.1.2" Condition="'$(UseSystemSqlite)' == 'True'" />
    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="2.1.2" Condition="'$(UseSystemSqlite)' != 'True'" />
  </ItemGroup>
```

---

I've force-pushed with the suggested change; this works fine on FreeBSD.

Since this adds Microsoft.Data.Sqlite.Core and bumps SQLitePCLRaw.provider.sqlite3 to 2.1.2 on the non-system SQLite build, could you check that this doesn't mess up something else (e.g. publish, etc.) on the other platforms?